clients/admin-ui/src/features/user-management/UserManagementRow.tsx

```diff
@@ -87,6 +93,23 @@ const UserManagementRow: React.FC<UserManagementRowProps> = ({ user }) => {
             </Badge>
           ))}
         </Td>
+        <Td pl={0} py={1} onClick={handleEditUser}>
+          <Badge
+            bg="gray.500"
+            color="white"
+            paddingLeft="8px"
```

Comment:
let's use the mappings provided by chakra here instead of giving exact pixel values

```suggestion
            paddingLeft="2"
```